src/Pages/InventoryItem/AddEditModal.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+
+const AddEditModal = (props) => {
+  return (
+    <Modal isOpen={props.open} toggle={props.toggle}>
+      <ModalHeader toggle={props.toggle}>{props.title}</ModalHeader>
+      <ModalBody className="add-item-body">
+        <form>
+          <label>Name:</label>
```

Comment:
homie i know you can use Array.map [like this](https://github.com/SCE-Development/Core-v4/blob/addd3e75b606aeefca6cfef47d7d4d56dc6810b8/src/Pages/LedSign/LedSign.js#L17)